scripts/install_plugin.sh

```diff
@@ -1,53 +1,87 @@
-#!/bin/sh -e
+#!/bin/sh
+set -e
 
-# Copied w/ love from the chartmuseum/helm-push :)
+# Function to print error message and exit
+error_exit() {
+    echo "$1" >&2
+    exit 1
+}
+
+# Function to validate command availability
+validate_command() {
+    command -v "$1" >/dev/null 2>&1 || error_exit "Required command '$1' not found. Please install it."
+}
 
-[ ! -z "$HELM_DEBUG" ] && set -x
```

Comment:
What happened to HELM_DEBUG logic? The intent there is to look at that env variable and add bash command verbosity.

Reply:
sorry it was just an unintentional deletion as I typed wrong number of lines to be dd-ed on vim... I have update the code now.